openedx/features/caliper_tracking/base_transformer.py

```diff
@@ -29,14 +30,35 @@ def __init__(self, event):
         self._add_referrer()
         self._add_extensions()
 
+    def _convert_datetime(self, current_datetime):
+        """
+        Convert provided datetime into UTC format
+
+        @param datetime: datetime string.
+        :return: UTC formatted datetime string.
+        """
+        utc_offset = current_datetime.utcoffset()
+        utc_datetime = current_datetime - utc_offset
+
+        formatted_datetime = utc_datetime.strftime(UTC_DATETIME_FORMAT)[:-3] + 'Z'
+        return formatted_datetime
+
     def _add_generic_fields(self):
         """
         Adds all of the generic fields to the event object
         """
         self.caliper_event.update({
             '@context': CALIPER_EVENT_CONTEXT,
             'id': uuid.uuid4().urn,
-            'eventTime': datetime.now().isoformat('T')[:-3] + 'Z'
+            'agent': self.event.get('agent'),
+            'edx_event_type': self.event.get('event_type'),
```

Comment:
Why is the key `edx_event_type` instead of `event_type`?

Reply:
fixed